### ISSUE TITLE
BUG: ADL iiif_print pdf split is not creating derivatives in staging only

### ISSUE THREAD

**Issue by ShanaLMoore** [scientist-softserv/adventist-dl#290]

**Comment by ShanaLMoore** [comment]
We attended Rob's office hr. 

The tesseract process was restarting sidekiq and losing the create derivative jobs. 

We will be increasing the worker's resources to resolve this.

**Comment by jeremyf** [comment]
> Tesseract needs a specific flag that we will also confirm exists, otherwise the processing speed is much slower.

@ShanaLMoore The following commit incorporates the ability to pass a tesseract flag: https://github.com/scientist-softserv/iiif_print/commit/32c01d9

**Comment by ShanaLMoore** [comment]
ok! 

But also, for future us, the issue we eventually saw was the worker container crashing due to limited resources.

**Comment by KatharineV** [comment]
tl;dr for the below:
I see derivatives in the action dropdown for multi-page PDFs uploaded to ADL staging today, but only for "new" works. Other aspects of the PDF split are not necessarily working as I expected, but derivatives are available in newly uploaded works[ like this one](https://adl.s2.adventistdigitallibrary.org/concern/published_works/20213893_critical_notes_upon_important_texts_of_scripture_relating_to_the?locale=en). 

I uploaded two PDFs as generic works to test the iiif_print PDF split. The files I used are attached here. The works are [Bookdrive by Atiz](https://adl.s2.adventistdigitallibrary.org/concern/generic_works/90695887-94cb-44a7-b2b1-b11dc33284b8?locale=en) and [Norfolk Island Weekly](https://adl.s2.adventistdigitallibrary.org/concern/generic_works/0aca20f5-234b-4130-9435-3a85476ea98f?locale=en). 

A large book import is running right now, which may be why I am stuck with this test. The two generic works are showing no files associated, despite the fact that files were uploaded. So, I can't test the PDF split on my generic works, but I'm seeing child works in the catalog search. These child works are associated with books in the large import that is in progress. Works with child works attached do show derivatives in the action drop down. Works without child works do not show derivatives in the action drop down. 

Example of a book with child works and derivatives: 
https://adl.s2.adventistdigitallibrary.org/concern/published_works/20213891_exposure_of_clerical_slander?locale=en 

Observations: 
On that book page, the PDF and the children are not showing in the viewer. I only see the thumbnail in the viewer. A keyword search in the viewer returns no results. In the second screenshot you see the results of my search for "slander." To me it looks like the PDF split is only applying to new books uploading to Staging for the first time--anything with an upload date of 03-13-2023. [This book's](https://adl.s2.adventistdigitallibrary.org/concern/published_works/20121634_de_natuur_en_verbinding_van_den_sabbath_volgens_het_vierde_gebodt?locale=en) files have an upload date in January, and while the importer says that it brought this work in again today, the work hasn't updated with anything new. Note the Items at the bottom of the page, which all have January upload dates and still list a broken PDF. **Derivatives are not available**. 

![Image](https://user-images.githubusercontent.com/118196520/224794441-f3b2bdaf-3793-446f-8b7f-974e7219a650.png)




![Image](https://user-images.githubusercontent.com/118196520/224795088-dd3e706f-3b60-4dfb-b38e-4e7dd3e4502a.png)




[bookdrive-n-brochure.pdf](https://github.com/scientist-softserv/adventist-dl/files/10960481/bookdrive-n-brochure.pdf)



[Norfolk Island Weekly 27 August 1943.pdf.pdf](https://github.com/scientist-softserv/adventist-dl/files/10960482/Norfolk.Island.Weekly.27.August.1943.pdf.pdf)

**Comment by KatharineV** [comment]
One of my generic works from yesterday has finished rendering, and now I can confirm that derivatives are created, they have thumbnails, the action drop down works, and the pages are OCR'd. 

Derivatives with thumbnails and action drop down:

![Image](https://user-images.githubusercontent.com/118196520/225053012-507a0658-c962-40d8-9781-0aca79495bd2.png)

Beautiful OCR search results:

![Image](https://user-images.githubusercontent.com/118196520/225053108-f986e9ee-becd-4a35-ae98-c36ae311484e.png)

**Comment by KatharineV** [comment]
Works as expected. Looks great. Thanks!
